packages/client/src/components/app/blocks/CardsBlock.svelte

```diff
@@ -126,7 +126,7 @@
             order={1}
           >
             {#if enrichedSearchColumns?.length}
-              {#each enrichedSearchColumns as column, idx}
+              {#each enrichedSearchColumns as column, idx (column.name)}
```

Comment:
Just curious what this does / is for?

Reply:
This is the fix for my second bullet point! Adding a parameter in parenthesis after the `as` in a svelte `each` statement defines how you'd like to key your items - it's basically saying "when this value changes, fully remount the component".
By using the name property, I'm basically saying that when a certain component instance receives a different name, fully remount it because it's a different field. By default svelte will reuse component instances and simply pass in different props when the number of items in an `each` loop changes, but this is only bug-free if your components are fully atomic and can handle it.
Basically adding this ensures that you never get that issue where you see "stale" search fields. The fields you see will now always be the correct ones. 
Hope that explanation helps!

Reply:
Thanks for the explanation - makes sense!